### DISCUSSION TITLE
Let me search my contributions (issues, pull requests, comments, ...)

### DISCUSSION THREAD

**Discussion by joeytwiddle** [community/community#63339] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

I tend to make a lot of small contributions to various projects all over GitHub, often just issues.

Sometimes I would like to go back and find an issue I was involved in years ago. I might not know the exact repo name, but I remember it belonged to the `jitsi` organisation, and the issue contained `*screen*`. I would like to search for my old comment, to find the issue again.

Or perhaps I want an overview of all the pull requests I have made, in all repositories, or perhaps in all repositories matching `*react*`. (Because I wanted to show some nice PRs to a HR recruiter.)

I'm afraid the Contributions page isn't really sufficient for this. It only shows the most recent things (which I haven't forgotten yet) and the search is manual (and loads a very limited amount of history after each click).

What I would really like is an advanced search, like we have in other areas of GitHub, but specifically for my contributions.

Amusingly, when I asked Bard if it was possible to do this, it hallucinated what a good solution might look like! I'll paraphrase below:

> On your profile page, click the link to your Contributions page, and type in the search bar:
>
> `is:issue commenter:me repo:jitsi/* screen`
>
> For pull requests to React libraries, which were made a while ago:
>
> `is:pr author:me repo:*react* before:31/12/2021`

A feature like this could help me to deep dive into my past activity.

(It could also let us deep dive into the history of other users, which they may or may not find desirable. Perhaps there could be an opt-in or opt-out for this.)

**Comment by mdnurulmomen** [comment, marked as answer]
First you may search it from the activity section at your profile overview page. 
Alternatively, you may search it from the icons at right-top corner (left side of profile-picture) of your profile page. 
I'm attaching a screen-shot for your better understanding.
![Screenshot 2023-08-10 113212](https://github.com/community/community/assets/46340880/7d628cc6-fa37-4e01-826f-491cd34323b6)

**Reply by mdnurulmomen** [reply]
![Screenshot 2023-08-10 113212](https://github.com/community/community/assets/46340880/2a5d07a2-fb7f-46f7-beac-f35345189043)

I'm marking the corner with red line. You may search your previous issues, pull-requests and many more activities there shortly.

**Comment by joeytwiddle** [comment]
Thanks so much!

![Screenshot_20230812_165030](https://github.com/community/community/assets/911799/4777a600-0bc3-4093-b424-ef563d432805)

Well, I feel a bit silly for not noticing those sooner.

When were they added to GitHub?
